UPGRADE.md

```diff
@@ -1,3 +1,16 @@
+### UPGRADE FROM 0.16.1 TO 0.17.0
+
+Invoices are now saved on the server during their generation (by default, when the order is paid).
+
+1. `Sylius\InvoicingPlugin\Creator\InvoiceCreator` class has 2 more dependencies: `InvoicePdfFileGeneratorInterface $invoicePdfFileGenerator`
```

Comment:
I am wondering if it will more clear if we show all constructor parameters instead of describing what have we changed.

Reply:
🤷‍♂️ both works for me 😄